cookbook/deployment/platformsh.rst

```diff
@@ -98,7 +98,9 @@ Configure Database Access
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Platform.sh overrides your database specific configuration via importing the
-following file (it's your role to add this file to your code base)::
+following file (it's your role to add this file to your code base):
+
+.. code-block:: php
```

Comment:
the previous code was correct (`::` is a shortcut for a PHP code block). We always prefer to use it instead of the long version for PHP code blocks.